spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -310,6 +310,130 @@ helm install --name my-release \
 ----
 ====
 
+[[spring-cloud-skipper-integration]]
+=== Spring Cloud Skipper Integration
+
+Skipper is a tool that allows you to discover Spring Boot applications and manage their lifecycle on multiple Cloud Platforms. 
+You can use Skipper standalone or integrate it with Continuous Integration pipelines to help achieve Continuous Deployment 
+of applications. For more details, review the link:https://docs.spring.io/spring-cloud-skipper/docs/{skipper-core-version}/reference/htmlsingle/#overview[reference guide]
+for a complete overview and the feature capabilities.
+
+Before we begin setting up Skipper to use with Spring Cloud Data Flow, let's review the basics by understanding foundational 
+design by which the relevant infrastructure is provisioned in Kubernetes. The tailormade 
+link:https://docs.spring.io/spring-cloud-skipper/docs/{skipper-core-version}/reference/htmlsingle/#tour-kubernetes[three-minute-tour for Kubernetes] 
+walks through the fundamentals.
+
+Next up, we will review the relevant artifacts to provision Spring Cloud Skipper in Kubernetes.
+
+==== Skipper Deployment
+[source,yaml,options=nowrap]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: skipper
+  labels:
+    app: skipper
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: skipper
+    spec:
+      containers:
+      - name: skipper
+        image: springcloud/spring-cloud-skipper-server:1.0.0.BUILD-SNAPSHOT
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 80
+        resources:
+          limits:
+            cpu: 1.0
+            memory: 1024Mi
+          requests:
+            cpu: 0.5
+            memory: 640Mi
+        env:
+        - name: SPRING_APPLICATION_JSON
+          value: "{\"spring.cloud.skipper.server.platform.kubernetes.accounts.minikube.environmentVariables\" : \"SPRING_RABBITMQ_HOST=${RABBITMQ_SERVICE_HOST},SPRING_RABBITMQ_PORT=${RABBITMQ_SERVICE_PORT}\"}"
```

Comment:
It would be better if we disable `local` platform by setting `spring.cloud.skipper.server.enableLocalPlatform` to `false` to avoid accidental deployment using `local` deployer when using Kubernetes etc.,

Reply:
Good point. @markpollack thought we could add `default` only to the local run of skipper-server. Not sure how we would differentiate that, though (?), we have only one server variant. I can disable it o'wise. And yes, the following table can be adapted to this change.